packages/rspack/src/rspackOptionsApply.ts

```diff
@@ -189,6 +190,11 @@ export class RspackOptionsApply {
 		if (options.builtins.devFriendlySplitChunks) {
 			options.optimization.splitChunks = undefined;
 		}
+
+		if (options.optimization.splitChunks) {
+			new SplitChunksPlugin(options.optimization.splitChunks).apply(compiler);
```

Comment:
My suggest is add a BuiltinPlugin::OldSplitChunksPlugin and register it based on experiments.newSplitChunks